cypress/support/step_definitions/event.js

```diff
@@ -0,0 +1,17 @@
+import { When } from '@badeball/cypress-cucumber-preprocessor';
+import nunjucks from 'nunjucks';
+
+When('I click on {string}', (selector) => cy.get(selector).click());
+
+When('I drag {string} onto {string}', (templateOriginSelector, templateDestinationSelector) => {
+  const originSelector = nunjucks.renderString(templateOriginSelector, cy.context);
+  const destinationSelector = nunjucks.renderString(templateDestinationSelector, cy.context);
+
+  cy.drag(originSelector, destinationSelector);
+});
+
+When('I drag {string} to {int},{int}', (templateSelector, templateX, templateY) => {
```

Comment:
if you write `templateX` please add `const x = nunjucks.renderString(templateX, cy.context);`
but if you don't please rename templateX to x
same for templateX